client-src/elements/chromedash-attachments.ts

```diff
@@ -0,0 +1,102 @@
+import {LitElement, css, html, nothing} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {createRef, ref, Ref} from 'lit/directives/ref.js';
+import {styleMap} from 'lit-html/directives/style-map.js';
+import {ChromedashTextarea} from './chromedash-textarea.js';
+import {URL_REGEX} from './form-field-specs.js';
+
+const DEFAULT_PLACEHOLDER = 'https://...\nhttps://...';
+const DEFAULT_SPLIT_PATTERN = String.raw`\s+`;
+
+@customElement('chromedash-attachments')
+export class ChromedashAttachments extends LitElement {
+  textareaRef: Ref<ChromedashTextarea> = createRef();
+
+  @property({type: String})
+  name;
+  @property({type: String})
+  value;
+  @property({type: String})
+  pattern;
+  @property({type: String})
+  chromedash_single_pattern = URL_REGEX;
+  @property({type: String})
+  chromedash_split_pattern = DEFAULT_SPLIT_PATTERN;
+
+  // Note: Form element components cannot have their own styles.
```

Comment:
This comment doesn't belong here?

Reply:
It's in the location where we would normally have the CSS styles.  In fact, I spent some time adding lit-css styles, only to realize that they are ignored.   I could put the comment on the next set of lines that avoids creating a renderRoot or on the lines that do the styleMap, but this is the location that would have helped me avoid wasted time.